base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -238,10 +238,10 @@ BEGIN
     IDS_ANGLE "Unghiul trebuie să fie între -89 și 89."
     IDS_LOADERRORTEXT "Fișierul %s nu poate fi încărcat."
     IDS_ENLARGEPROMPTTEXT "Imaginea din memorie este mai mare decât suprafața curent disponibilă în aplicație.\nDoriți să redimensionați suprafața de desen disponibilă?"
-    IDS_BOLD "Bold"
-    IDS_ITALIC "Italic"
+    IDS_BOLD "Îngroșat"
+    IDS_ITALIC "Cursiv"
     IDS_UNDERLINE "Subliniat"
     IDS_VERTICAL "Vertical"
-    IDS_PRINTRES "%d x %d pixel/cm"
-    IDS_CANTPASTE "Failed to paste from the clipboard. The data format is either incorrect or not supported."
+    IDS_PRINTRES "%d x %d pixeli/cm"
+    IDS_CANTPASTE "Nu s-a putut lipi din clipboard. Formatul de date este fie incorect, fie nesuportat."
```

Comment:
Please use passive mood instead of reflexive one. Please see [Romanian translation notes](https://raw.githubusercontent.com/reactos/reactos/master/media/doc/Romanian%20translation%20notes.txt), point №20.